_layouts/base.html

```diff
@@ -1,6 +1,7 @@
 ---
 layout: default
 ---
+{% assign keywords = "google,asylo,opensource framework,flexible framework,opensource application,asylo-site" %}
```

Comment:
fitting "enclave" in the list would be good.

Reply:
done